frontend/src/metabase/components/EntityItem/EntityItem.jsx

```diff
@@ -45,9 +45,7 @@ function EntityIconCheckBox({
   return (
     <EntityIconWrapper
       isPinned={pinned}
-      model={item.model}
```

Comment:
removed inexistent props. This component is a styled component that styles another styled component and both components doesn't have either `model` or `rounded` prop. I suspect this has been left out from a refactoring some time ago.

https://github.com/metabase/metabase/blob/f2d279cd88bc59b6dd269dba2e625b44e65f093e/frontend/src/metabase/components/EntityItem/EntityItem.styled.tsx#L30-L33

https://github.com/metabase/metabase/blob/f2d279cd88bc59b6dd269dba2e625b44e65f093e/frontend/src/metabase/components/IconButtonWrapper/IconButtonWrapper.tsx#L3